scipy/_lib/_testutils.py

```diff
@@ -20,9 +20,34 @@ class FPUModeChangeWarning(RuntimeWarning):
 
 class PytestTester:
     """
-    Pytest test runner entry point.
-    """
+    Run tests for this namespace
+
+    ``scipy.test()`` runs tests for all of SciPy, with the default settings.
+    When used from a submodule (e.g., ``scipy.cluster.test()``, only the tests
+    for that namespace are run.
+
+    Parameters
```

Comment:
This section might be misleading as it's not the parameters for the init but the call method. Maybe adding a small example or a one liner explanation would help?

Reply:
I considered that and actually started writing a simple example using `scipy.test` - before realizing that that would be extraordinarily expensive if the doctests are run:)

Reply:
Right 👍